R/map.R

```diff
@@ -125,50 +130,70 @@
 #'   map(\(df) lm(mpg ~ wt, data = df)) |>
 #'   map(summary) |>
 #'   map_dbl("r.squared")
-map <- function(.x, .f, ..., .progress = FALSE) {
-  map_("list", .x, .f, ..., .progress = .progress)
+#'
+#' @examplesIf interactive()
+#' # run in interactive sessions only as spawns additional processes
+#'
+#' # To use parallelized map, set daemons (number of parallel processes) first:
+#' daemons(2)
```

Comment:
What happens if you don't set daemons?

....

Oh I see you describe the later.

Here it might be worth a comment that in user code you'd normally leave the daemons running throughout your session? (I assume that's the case?)

Reply:
Yes, setting daemons once per session is usually the idea. Added in [84d6b63](https://github.com/tidyverse/purrr/pull/1163/commits/84d6b6356a8ba7712bf6e54a23aeb3bd3bbc9506).

Reply:
Also I've updated the behaviour so that there is now an error with a suggestion if daemons are not set. I think this is what you had in mind.